spec.md

```diff
@@ -56,4 +48,43 @@ The high-level components of the spec include:
 * Signatures that are based on signing image content address (optional OCI layer)
 * Naming that is federated based on DNS and can be delegated (optional OCI layer)
 
+## Actions
+
+The specified [components](#components) MAY be manipulated with actions.
+
+* **pack**: Create a blob from the local filesystem (or other storage).
+* **unpack**: Update the local filesystem (or other storage) based on data found in a blob.
+* **validate**: Verify that blobs [comply](#notational-conventions) with the specification for their [media type](media-types.md).
+    Blobs can be validated on their own (**intra-blob validation**) or recursively including [referenced](descriptor.md) ancestors (**recursive validation**).
+* **translate**: Convert a blob between similar formats.
+    Translation may be **lossy** or **lossless** depending on how well the formats align.
+
+## Publishing Images
+
```

Comment:
This section is talking about packing image. If the title had better to be `packing image`?

Publishing might be relative to distribution.